src/lib/drivers/extension.js

```diff
@@ -80,17 +80,34 @@ class ExtensionDriver extends Driver {
     this._listeners[eventName].push(cb);
   }
 
+  /**
+   * Bind a one-time listener for protocol events. Listener is removed once it
+   * has been called.
+   * @param {!string} eventName
+   * @param {function(...)} cb
+   */
+  once(eventName, cb) {
+    // Create a replacement listener that will immediately remove itself after calling cb once.
+    const cbGuard = function() {
```

Comment:
I left a note on this before, but lost with the commit shuffling. `arguments` can't be used in arrow functions, so can't use them here. Rest parameters is the way to handle this situation in arrow functions, but they aren't available in node until node 6